Library/Formula/tiff2png.rb

```diff
@@ -2,25 +2,15 @@
 
 class Tiff2png < Formula
   homepage 'http://www.libpng.org/pub/png/apps/tiff2png.html'
-  url 'ftp://ftp.simplesystems.org/pub/libpng/png/applications/tiff2png/tiff2png-0.91.tar.gz'
-  sha1 '3a23abaaadbed8f3d13b88241257fe2078eb61fd'
-  revision 1
+  url 'https://people.xiph.org/~giles/tiff2png/tiff2png-0.92.tar.gz'
+  sha256 'bf743ae5b231d690202eb8a552f173619815533172417aeb3f8a75e13315f0ed'
```

Comment:
Git's SHA1 is in a file downloaded from github over TLS--where it's protected by a SHA2 signature.

What's your actual objection to SHA-256.